docs/content/how-tos/rules/FL0080.md

```diff
@@ -0,0 +1,33 @@
+---
+title: FL0080
+category: how-to
+hide_menu: true
+---
+
+# UnnestedFunctionNames (FL0080)
+
+*Introduced in `0.21.8`*
+
+## Cause
+
+Missing "return" keyword inside async blocks when throwing exceptions.
```

Comment:
@webwarrior-ws when resolving conflicts, and when addressing my comments, please see your own diff before pushing again ^ (the whole diff)

Reply:
Put back original text in docs